<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHistoryDeferPosiSearchDlg</class>
 <widget class="QDialog" name="frmHistoryDeferPosiSearchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>历史延期持仓查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1041</width>
     <height>401</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_query">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:50px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>71</y>
      <width>911</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>合约代码</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_ed">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_page">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_output_excel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>输出到excel</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_st">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_code">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_first">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:50px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>|&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_option">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>设置显示列</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_num">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>开始日期</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_last">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
color:white;
font: 12pt &quot;宋体&quot;;
        border-radius: 3px;
        border: none;
		background: #2a3a57;
        height:25px;
        width:35px;
        margin-top:0px;
        margin-bottom:5px;
}

/*划过按钮*/
QPushButton:enabled:hover{
        background: #227dc3;
}

QPushButton:enabled:pressed{
        background: #227dc3;
}</string>
    </property>
    <property name="text">
     <string>&gt;|</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>结束日期</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
